source/reporting/Report database.rst

```diff
@@ -0,0 +1,248 @@
+ReportingService: Documentation of tables
+=========================================
+Documentation of existing tables per. 20 November 2020. Number of tables / columns documented: 30/446 (see Appendix for further details).
+
+[billing].[PaymentWithdrawals]
+----------------------------
+PaymentWithdrawals includes credit card debit attempts.
+
+[dbo].[Organizations]
+---------------------
+Information about organizations such as organization name, e-mail , phone and address.
+
+In order for the various tables to be as easy to read as possible, parts of this information are also found in many of the tables in the report layer . Here some may have many titles / organizations. The main point of this is to be able to structure different titles separately, but with different offers, invoice runs and subscription plans.
+
+[dbo].[ProductReportingGroups]
+------------------------------
+ProductReportingGroups are used to put products into different groups. For example, it can be used to make it clear that products A and B belong to report group 1, while products C, D and E belong to report group 2.
+
+
+[dbo].[Products]
+----------------
+Products provides an overview of all products that are available, and has a more detailed description of these.
+
+Special fields in Products:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+**Productcategory**: This is a grouping table for the report layer, which can be used to split the reporting into groups. You create a group, and then link products to the groups.
+
+ 
+
+[dbo].[ReportingGroups]
+-----------------------
+Report groups are defined to distinguish between different product types. Examples of this could be that one product is digital, another is on paper (physical), while a third is a combination of both digital and paper.
+ 
+
+[dbo].[SubscriberContacts]
+------------------------
+All contact addresses: Main addresses, billing addresses and temporary addresses.
+
+Special fields in SubscriberContacts:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+**IsPrimary**: Value 1 means the address is home address, 0 is billing address or temporary address.
+
+**Identification**: May contain information about organization number, customer reference and more. JSON functions must be used to retrieve each individual value in this type of field.
+ 
+
+[dbo].[Subscribers]
+-------------------
+Subscriber is a register of all subscribers and contains all contact information. This information is used for deliveries and invoicing. Examples of use are sending a PDF invoice to an e-mail address or sending a physical invoice to a postal address.
```

Comment:
Important that the contact information here is fetched from the primary address.

Reply:
Is this better? Subscriber is a register of all subscribers and contains all contact information from the primary address.

Reply:
Yes